src/lib.rs

```diff
@@ -2151,6 +2170,12 @@ impl Build {
         };
 
         let (sdk_prefix, sim_prefix, min_version) = match os {
+            Os::MacOs => (
+                "macosx",
+                "",
+                std::env::var("MACOSX_DEPLOYMENT_TARGET")
+                    .unwrap_or_else(|_| (if self.cpp { "10.9" } else { "10.4" }).into()),
```

Comment:
10.4 is older than Rust's deployment target. We default to 10.7, although this may change soon.